virtio_gen/src/virtio_blk.rs

```diff
@@ -1,46 +1,51 @@
-// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why did the copyright header disappear? Same for [virtio_net.h](https://github.com/firecracker-microvm/firecracker/pull/853/commits/a6a5183a3a5fc51e2e4967117302388911bc82a8#diff-c49b8bb107fdc75f81ce11681db47087).

Reply:
I don't think that we should have Copyright headers for autogenerated code because we just call bindgen, there is nothing else that we do to get these files. We should get more people involved to see what is the right approach for this case.